common/static/coffee/spec/discussion/discussion_spec_helper.coffee

```diff
@@ -317,7 +317,7 @@ browser and pasting the output.  When that file changes, this one should be rege
         <ul class="post-errors" style="display: none"></ul>
         <div class="forum-new-post-form-wrapper"></div>
         <% if (cohort_options) { %>
-        <div class="post-field">
+        <div class="post-field group-selector-wrapper<% if (!is_commentable_cohorted) { %> disabled<% } %>">
```

Comment:
Oh wow, this massive Underscore template in this Coffeescript file is pretty terrible. This template should be moved out of this file and into a separate file, but that is out of scope for this pull request. The change that you've made here is perfectly fine, but I'm just leaving this comment here as a reminder for me (or someone else) to come back here and move this template out at some point in a future clean-up pull request.